content/articles/2018-10-dplyr-0-8-0-release-candidate.Rmarkdown

```diff
@@ -0,0 +1,285 @@
+---
+title: dplyr 0.8.0 release candidate
+author: Romain François
+date: '2018-10-23'
+slug: dplyr-0-8-0-release-candidate
+description: > 
+  What you need to know about upcoming changes for dplyr 0.8.0.
+categories:
+  - package
+tags:
+  - dplyr
+  - tidyverse
+photo:
+  url: https://unsplash.com/photos/kU-WKSyTcp4
+  author: Pau Casals
+---
+
+```{r setup, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(collapse = T, comment = "#>")
+options(tibble.print_min = 4L, tibble.print_max = 4L)
+library(dplyr)
+```
+
+A new release of dplyr (0.8.0) is on the horizon, and since it is a major release, we'd love for the 
+community to try it out, give us some feedback and [report issues](https://github.com/tidyverse/dplyr/issues)
+before we submit to CRAN. This version represents about six months of development, making dplyr more
+respectful of factors and less surprising in its evaluation of expression. In this post, we'll highlight 
+the major changes, please see the [NEWS](https://github.com/tidyverse/dplyr/blob/master/NEWS.md) for a more 
+detailed description of changes. 
+
+Our formalised process for this release is captured in [this issue](https://github.com/tidyverse/dplyr/issues/3931)
+
+```{r, eval = FALSE}
+# install.packages("devtools")
+devtools::install_github("tidyverse/dplyr")
+```
+
+If needed, you can restore the release version by installing from CRAN:
+
+```{r, eval = FALSE}
+install.packages("dplyr")
+```
+
+# New grouping algorithm
+
+## Group creation
```

Comment:
It might be nice to add an anecdote about this closing the epic issue https://github.com/tidyverse/dplyr/issues/341 that dates back to 2014